### PR TITLE
Fix trivial bugs

### PR DESCRIPTION
**Pull request by PabstMirror**
- pass mag (if weapon == "" then CBA_fnc_compatibleMagazines will throw warning
- med-engine - add ace tag to debug line
- overheat, `_energyIncrement ` would be nil on first run because of inner private

### CODE REVIEW

File: addons/medical_engine/script_component.hpp [hunk 1]
```diff
@@ -19,11 +19,11 @@
 #include "\z\ace\addons\medical_engine\script_macros_config.hpp"
 
 #define PRELOAD_CLASS(class) \
-    diag_log format["Starting preload for ""%1""",class];\
+    diag_log text format["[ACE] (medical_engine) Starting preload for (%1)",class];\
```
Comment (jonpas) [inline addons/medical_engine/script_component.hpp]:
```suggestion
    INFO_1("Starting preload for (%1)",class);\
```

File: addons/medical_engine/script_component.hpp [hunk 2]
```diff
@@ -19,11 +19,11 @@
...
     [{\
         1 preloadObject _this;\
     }, {\
-        diag_log format ["Preload done for ""%1""",_this];\
+        diag_log text format ["[ACE] (medical_engine) Preload done for (%1)",_this];\
```
Comment (jonpas) [inline addons/medical_engine/script_component.hpp]:
```suggestion
        INFO_1("Preload done for (%1)",_this);\
```

File: addons/interaction/functions/fnc_canPassMagazine.sqf [hunk 3]
```diff
@@ -19,6 +19,7 @@
 params ["_player", "_target", "_weapon"];
 
 if (!GVAR(enableMagazinePassing)) exitWith {false};
+if (_weapon == "") exitWith {false};
```
Comment (dedmen) [inline addons/interaction/functions/fnc_canPassMagazine.sqf]: isEqualTo :smirk:

Comment (commy2) [reply addons/interaction/functions/fnc_canPassMagazine.sqf]:
```suggestion
if (_weapon isEqualTo "") exitWith {false};
```

**Pull request merged**